apps/entry/management/commands/fix_ahhs_data.py

```diff
@@ -0,0 +1,75 @@
+import csv
+import os
+import logging
+from decimal import Decimal
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from apps.entry.models import Figure
+from utils.common import round_half_up
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    help = "Fix AHHS household size data."
+
+    def add_arguments(self, parser):
+        parser.add_argument('csv_file_path', type=str, help="Path to the CSV file containing the data.")
+
+    # FIXME Use Bulk Manager
+    def patch_figure_household_from_csv(self, file_path):
+        change_in_repoted_value_figures = []
+        change_in_unit_figures = []
+        figure_id_not_found = []
+
+        with open(file_path, 'r') as file:
+            reader = csv.DictReader(file)
+            for row in reader:
+                fig_id = row['figure_id']
+                figure = Figure.objects.get(id=fig_id)
```

Comment:
This will raise exception
```suggestion
                figure = Figure.objects.filter(id=fig_id).first()
```